src/ontology/mondo-edit.obo

```diff
@@ -404503,13 +404544,13 @@ relationship: has_characteristic MONDO:0021152 {source="OMIMPS:148210"} ! inheri
 id: MONDO:0018782
 name: obsolete type 1 interferonopathy
 subset: gard_rare {source="GARD:21957"}
```

Comment:
should this be removed? There are actually probably a lot of cases where obsolete tags are still on obsolete terms, maybe we should create a separate ticket for this (if we don't already have one)

Reply:
YES!!! Thank you! I thought about it then got distracted. I will do this right now! Thank you for catching this.